types/jss/jss-tests.ts

```diff
@@ -46,6 +47,9 @@ const rule = attachedStyleSheet.addRule('dynamicRule', { color: 'indigo' });
 rule.prop('border-radius', 5).prop('color'); // $ExpectType string
 attachedStyleSheet.classes.dynamicRule; // $ExpectType string
 
+const generateClassName = createGenerateClassName();
+const dynamicRuleClassName = generateClassName(rule, attachedStyleSheet);
```

Comment:
you should not be using generateClassName, its a function that JSS is using, its interface should be private

Reply:
I mean the interface is not really private, since you are allowed to implement your own class name generator, but generating the class name from it is what jss does, so I see no point in showing how to use it as a user